phill.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title></title>
+</head>
+<body>
+  <h1> phill cheng
+    <img src="https://media.giphy.com/media/l2JJGu5LiSkdtp2Du/giphy.gif"</img>
+  </h1>
+  <p>
+    Slack username: P.J.C.1.
```

Comment:
Please surround each line of text with an individual `<p>` tag.

Change to the following:
```html
  <p>Slack username: P.J.C.1.</p>
  <p>GitHub username: PJC-1</p>
```